examples/react-native/src/App/App.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-import { AuthenticatorDemo } from '../features/Authenticator';
+// import Example from '../features/Authenticator/Demo/Example';
+import Example from '../features/Authenticator/Styles/Example';
 // import { Demo as InAppDemo } from '../features/InAppMessaging';
```

Comment:
Moving forward we will use this structure for RN `Example` apps so we have a consistent import structure for using .env variables to dynamically load `Example` apps